### PR TITLE
Add type hints into optuna/dashboard.py, optuna/integration/__init__.py

### PR DESCRIPTION
**Pull request by airyou**
## Motivation

Please refer to the Issue #711 .

## Description of the changes

I added type hints into optuna/dashboard.py, optuna/integration/__init__.py

### CODE REVIEW

File: optuna/dashboard.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import collections
 import threading
 import time
+from typing import Any  # NOQA
+from typing import Dict  # NOQA
+from typing import List  # NOQA
+from typing import Optional  # NOQA
```
Comment (HideakiImamura) [inline optuna/dashboard.py:7]: We can just remove `# NOQA`. Could you remove them?

**Review by HideakiImamura — COMMENTED** [review]
Thanks for your contribution! LGTM except for a minor comment.

File: optuna/dashboard.py (hunk 1 above)
Comment (hvy) [reply optuna/dashboard.py:7]: Seems to apply to the import of `Any` in `optuna/integration/__init__.py` as well. 🙂

**Review by hvy — CHANGES_REQUESTED** [review]
Looks great except for a nit which I left [here](https://github.com/optuna/optuna/pull/1653/files#r473578563).

File: optuna/dashboard.py (hunk 1 above)
Comment (airyou) [reply optuna/dashboard.py]: Thank you for your review.
I fixed the code as you advised.

**Review by HideakiImamura — APPROVED** [review]
Thanks for your swift actions! LGTM!

**Comment by airyou** [comment]
Sorry, I miss operate about review request.

**Pull request merged**